apache-hadoop-mixin/dashboards/hadoop-datanode-overview.libsonnet

```diff
@@ -338,10 +340,22 @@ local datanodeLogsPanel = {
               allValues='.+',
               sort=1
             ),
+            template.new(
+              'cluster',
+              promDatasource,
+              'label_values(hadoop_datanode_ramdiskblocksevictedwithoutread{%(hadoopSelector)s}, cluster)' % $._config,
```

Comment:
Should use `multiClusterSelector` instead of `hadoopSelector` for the `Cluster` variable.